src/org/omegat/Bundle.properties

```diff
@@ -1402,7 +1402,7 @@ ABOUTDIALOG_COPYRIGHT=Copyright \u00a9 2000-2016 Keith Godfrey, Maxym Mykhalchuk
+    OmegaT is a free and open source multiplatform Computer Assisted Translation tool with fuzzy matching, translation memory, keyword search, glossaries, and translation leveraging into updated projects.\n\nOmegaT is the original work of Keith Godfrey.\nDidier Briel is the OmegaT project manager.\n\nCode has been contributed by: \nZoltan Bartko, Volker Berlin, Didier Briel, Kim Bruning, Alex Buloichik (lead developer), Thomas Cordonnier, Sandra Jean Chua, Enrique Est\u00e9vez Fern\u00e1ndez, Martin Fleurke, Wildrich Fourie, Tony Graham, Phillip Hall, Jean-Christophe Helary, Chihiro Hio, Thomas Huriaux, Hans-Peter Jacobs, Kyle Katarn, Piotr Kulik, Ibai Lakunza Velasco, Guido Leenders, Aaron Madlon-Kay, Fabi\u00e1n Mandelbaum, Manfred Martin, Adiel Mittmann, Hiroshi Miura, John Moran, Maxym Mykhalchuk, Arno Peters, Henry Pijffers, Briac Pilpr\u00e9, Tiago Saboga, Andrzej Sawu\u0142a, Benjamin Siband, Yu Tang, Rashid Umarov, Antonio Vilei, Ilia Vinogradov, Martin Wunderlich and Michael Zakharov\n\nOther contributions are detailed in the readme.txt file\n\nOmegaT uses the following libraries:\n  HTMLParser 1.6 by Somik Raha, Derrick Oswald and others (LGPL License)\n  VLDocking Framework 3.0.5-SNAPSHOT (LGPL License)\n  Hunspell by L\u00e1szl\u00f3 N\u00e9meth and others (LGPL License)\n  JNA by Todd Fast, Timothy Wall and others (LGPL License)\n  Swing-Layout 1.0.4 (LGPL License)\n  Jmyspell 2.1.4 (LGPL License)\n  SVNKit 1.8.5 (TMate License)\n  Sequence Library (Sequence Library License)\n  ANTLR 3.4 (ANTLR 3 License)\n  SQLJet 1.1.10 (GPL License)\n  JGit (Eclipse Distribution License)\n  JSch (JSch License)\n  Base64 (public domain)\n  Diff (GPL)\n  trilead-ssh2-1.0.0-build217 (Trilead SSH license)\n  Lucene (Apache License 2.0)\n  TinySegmenter (public domain)\n  commons-*.jar (Apache License 2.0)\n  jwordsplitter-3.4.jar (Apache License 2.0)\n  LanguageTool.jar (LGPL License)\n  morfologik-*.jar (Morfologik license)\n  segment-1.4.1.jar (Segment License)\n  lucene-gosen-2.0.2-ipadic.jar (LGPL License)\n  pdfbox-app-1.8.1.jar (Apache License 2.0)\n  KoreanAnalyzer-3x-120223.jar (Apache License 2.0)\n  SuperTMXMerge-for_OmegaT.jar (LGPL License)\n  groovy-all-2.2.2.jar (Apache Licence 2.0)\n  slf4j-*.jar (MIT License)\n  juniversalchardet-1.0.3.jar (GPL)\n  DictZip from JDict (GPL)\n trie4j-0.9.1.jar (Apache License 2.0)
```

Comment:
Each continuing line should start with two spaces, but you only have one for `trie4j`.